### ISSUE TITLE
Terminology in code

### ISSUE THREAD

**Issue by annayqho** [annayqho/AnnaCannon#19]
Make it consistent with the paper. From what I can tell, this is the final list. 
- "stellar labels"
- "reference objects"
- "survey objects" 
- "training step"
- "test step"
- "label transfer"

**Comment by davidwhogg** [comment]
Not only should the words match, the symbols should too -- like M, m, N, n, theta, ell, etc...
